wiki/Rules/Content_Voting_Process/id.md

```diff
@@ -4,21 +4,21 @@ Konten-konten yang diajukan kepada para anggota [Beatmap Nominator](/wiki/People
 
 ## Proses kajian
 
-1. Pada mulanya, para mapper, modder, atau pengguna lainnya yang merasa menemukan suatu hal/konten pada beatmap yang disinyalir melanggar [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations) melaporkan hasil temuan mereka melalui situs web Beatmap Nominator.
-2. Laporan yang masuk tersebut akan diperiksa keabsahannya oleh NAT, di mana NAT kemudian akan membuka “berkas kasus” (*content case*) untuk konten yang bersangkutan. Masing-masing anggota BN, GMT, dan NAT kemudian akan menyatakan suaranya secara terpisah terkait apakah konten yang bersangkutan layak guna atau tidak.
-3. Pada saat berkas kasus yang bersangkutan masih terbuka, beatmap-beatmap yang dilaporkan tidak dapat dinominasikan ataupun di-*qualify*. Beatmap-beatmap yang sebelumnya sudah berstatus Qualified akan didiskualifikasi dengan segera.
-4. Berkas kasus yang bersangkutan akan secara otomatis ditutup apabila dalam rentang waktu 3 hari tidak ada suara baru yang masuk atau apabila 7 hari telah berlalu.
+1. Seorang mapper, modder, atau pengguna melaporkan suatu konten visual pada beatmap yang disinyalir melanggar [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations) melalui situs web Beatmap Nominator.
+2. Laporan tersebut diperiksa keabsahannya oleh NAT, dan — kecuali pada kasus-kasus yang jelas melanggar aturan — sebuah “berkas kasus” (*content case*) untuk konten yang bersangkutan dibuka untuk dilakukan pemungutan suara. Semua anggota BN, GMT, dan NAT berhak memberikan suaranya secara terpisah.
```

Comment:
```suggestion
2. Laporan tersebut diperiksa keabsahannya oleh NAT, dan—kecuali pada kasus-kasus yang jelas melanggar aturan—sebuah “berkas kasus” (*content case*) untuk konten yang bersangkutan dibuka untuk dilakukan pemungutan suara. Semua anggota BN, GMT, dan NAT berhak memberikan suaranya secara terpisah.
```
im kinda stupid lol. the spaces aren't supposed to be there

Reply:
o the spaces are meant to be there actually because they're em dashes (https://www.lexico.com/grammar/dash-and-em-dash) instead of a regular dash character

Reply:
em dashes also don't have spaces in between them